blueprints/addon/index.js

```diff
@@ -106,6 +109,12 @@ module.exports = {
     return new FileInfo(options);
   },
 
+  beforeInstall() {
```

Comment:
Should this be in the module-unification-addon blueprint instead of the addon one?

Reply:
Done by https://github.com/ember-cli/ember-cli/pull/8323/commits/89e76fc94fe98f7b6db549ddea8d94da6118594e